ledger/complete/checkpoint_benchmark_test.go

```diff
@@ -145,7 +145,6 @@ func BenchmarkLoadCheckpointAndWALs(b *testing.B) {
 			return err
 		},
 		func(rootHash ledger.RootHash) error {
-			forest.RemoveTrie(rootHash)
```

Comment:
With removal of removal, maybe we can get out of it completely, remove `WALDelete` (and `WALUpdate` since now we have only one operation, and all related code

Reply:
I think we need it for reading old WALs when we spork, we could get rid of it after next spork.